packages/transformers/react-refresh-babel/src/ReactRefreshBabelTransformer.js

```diff
@@ -0,0 +1,117 @@
+// @flow
+
+import semver from 'semver';
+import {Transformer} from '@parcel/plugin';
+import {relativeUrl} from '@parcel/utils';
+import SourceMap from '@parcel/source-map';
+import {transformFromAst} from '@babel/core';
+import generate from '@babel/generator';
+import {parse} from '@babel/parser';
+
+function shouldExclude(asset, options) {
+  return (
+    !asset.env.isBrowser() ||
+    !options.hot ||
+    !asset.isSource ||
+    asset.filePath.includes('packages/runtimes')
```

Comment:
There should be a way to distinguish "user" assets (`isSource` isn't enough) from node_modules / generated ones.

Reply:
Won't this not work outside the monorepo?

Reply:
Yes, this is a hack for testing until I find a better way.